Sombra.Web/wwwroot/js/chart.js

```diff
@@ -105,3 +105,5 @@ var myChart = new Chart(ctx, {
         }
     }
 });
+
```

Comment:
undo changes when a file hasnt really been changed next time you make a pull request